README.md

```diff
@@ -79,9 +79,28 @@ At the moment, your app is just getting data about one GitHub user. But what if
 Try and sketch out a plan!
 
 
-### Challenge 3: code coverage
+### Challenge 3: check code coverage of tests
+
+How could you test your app? Try and write some tests for the different parts of your application then check code coverage of those tests.
+
+Do you remember that you have been introduced to ```code coverge``` in research afternoon in week2 ?
```

Comment:
typo? (coverge)

Reply:
Sorry, somehow I completely missed the notification for this. 
I am sharing [action points](https://github.com/foundersandcoders/master-reference/issues/630#issuecomment-331800134) to be made in the meeting with London, Gaza and Naz Campus.